### PR TITLE
Support multiple yml files in the postgres_exporter startup options (tls support)

### PR DESCRIPTION
**Pull request by keithf4**
# Description  

Testing TLS support for postgres_exporter and ran into a parsing error when there are multiple *.yml files in the startup $OPT. Try to only grab the first file named after the query_path.

Fixes #  

Depends on #  

## Type of change  
Please check all options that are relevant  
- [x] Bug fix (change which fixes an issue)  
- [ ] Feature (change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [ ] Documentation update only  

# How Has This Been Tested?  
**Tested Configuration**:  
- [ ] Ansible, Specify version(s):  
- Installation method:  
    - [x] Binary install from source, version:  postgres_exporter 0.10.0
    - [ ] OS package repository, distro, and version:  
    - [ ] Local package server, version:  
    - [ ] Custom-built package, version:  
    - [ ] Other:  
- [x] PostgreSQL, Specify version(s):  14
- [ ] docs tested with hugo version(s):  

Tested with playbook(s):  

# Checklist:  
- [x] My changes generate no new lint warnings  
- I have made corresponding changes to:  
    - [ ] the documentation  
    - [ ] the release notes  
    - [ ] the upgrade doc

### CODE REVIEW

**Review by hunleyd — APPROVED** [review]

File: postgres_exporter/linux/crunchy-postgres-exporter@.service [hunk 1]
```diff
@@ -15,7 +15,7 @@ After=network.target
 PermissionsStartOnly=true
 User=ccp_monitoring
 EnvironmentFile=/etc/sysconfig/%i
-ExecStartPre=/bin/bash -c "set -o pipefail; cat $QUERY_FILE_LIST | sed -e 's/#PG_STAT_STATEMENTS_THROTTLE_MINUTES#/${PG_STAT_STATEMENTS_THROTTLE_MINUTES}/g' -e 's/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES}/g' -e 's/#PG_STAT_STATEMENTS_LIMIT#/${PG_STAT_STATEMENTS_LIMIT}/g' > $$(echo $OPT | sed 's/.*--extend.query-path=\(.*\.yml\).*/\1/')"
+ExecStartPre=/bin/bash -c "set -o pipefail; cat $QUERY_FILE_LIST | sed -e 's/#PG_STAT_STATEMENTS_THROTTLE_MINUTES#/${PG_STAT_STATEMENTS_THROTTLE_MINUTES}/g' -e 's/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES}/g' -e 's/#PG_STAT_STATEMENTS_LIMIT#/${PG_STAT_STATEMENTS_LIMIT}/g' > $$(echo $OPT | sed 's/.*--extend.query-path=\([^\s]*\.yml\).*/\1/')"
```
Comment (sharmay) [inline postgres_exporter/linux/crunchy-postgres-exporter@.service:18]: Testing just this sed with echo is not giving me correct restults:

Commands:
```
export OPT=" --web.listen-address=:9187 --extend.query-path=/etc/postgres_exporter/14/queries_golf_global.yml --disable-default-metrics --disable-settings-metrics --web.config.file=/var/lib/ccp_monitoring/postgres_exporter_web_config_file_golf_file.yml"
echo $OPT | sed 's/.*--extend.query-path=\([^\s]*\.yml\).*/\1/'
```

Results:
```
--web.listen-address=:9187 --extend.query-path=/etc/postgres_exporter/14/queries_golf_global.yml --disable-default-metrics --disable-settings-metrics --web.config.file=/var/lib/ccp_monitoring/postgres_exporter_web_config_file_golf_file.yml
```

Comment (keithf4) [reply postgres_exporter/linux/crunchy-postgres-exporter@.service:18]: Weird that it works in the service file then. Any suggestions for a better fix?

Comment (keithf4) [reply postgres_exporter/linux/crunchy-postgres-exporter@.service:18]: This seems to work?
```
echo $OPT | sed 's/.*--extend.query-path=\(.*\.yml\) .*/\1/'
/etc/postgres_exporter/14/queries_golf_global.yml```

Comment (keithf4) [reply postgres_exporter/linux/crunchy-postgres-exporter@.service:18]: Did confirm that last change works just adding a space after the `(.*\.yml\)`

Least that seems to work consistently both in and out of the service file. Still open to better suggestions.

Comment (keithf4) [reply postgres_exporter/linux/crunchy-postgres-exporter@.service:18]: Yogesh pointed out that last change doesn't work if --extend.query-path is the last option

Comment (keithf4) [reply postgres_exporter/linux/crunchy-postgres-exporter@.service:18]: More testing seems to show that the below seems to work (replace the `\s` with my initial change with a plain space ` `)
```
sed 's/.*--extend.query-path=\([^\ ]*\.yml\).*/\1/'
```

**Review by sharmay — APPROVED** [review]

**Review by sharmay — APPROVED** [review]

**Review by pgguru — APPROVED** [review]

**Review by hunleyd — APPROVED** [review]

**Pull request merged**